errai-data-binding/src/main/java/org/jboss/errai/databinding/rebind/DataBindingUtil.java

```diff
@@ -467,4 +467,12 @@ public static boolean isBindableType(final MetaClass type) {
 
     return bindableTypes;
   }
+
+  private static boolean validateWildcard(MetaClass bindable) {
+    if(bindable.isFinal()) {
```

Comment:
Please fix the formatting so there is space between the `if` and first parenthesis.